spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -16,9 +16,10 @@
   end
 
   describe "#valid?" do
+    let(:status) { :published }
     let(:state) { :published }
```

Comment:
Should this be removed if we're now using `:status`?

Reply:
Nope as we need to be able to set the exact state for this test - https://github.com/alphagov/content-publisher/pull/1941/files#diff-8372787ef5224cbd5801d35735fc6dd4R115